examples/clerk/README.md

```diff
@@ -1,3 +1,32 @@
-# Minimal Jazz Auth Clerk Example
+# Clerk authentication example
 
-This is a minimal example of how to use Clerk authentication with Jazz.
+Live version: https://clerk-demo.jazz.tools/
+
+## Installing & running the example locally
+
+(this requires `pnpm` to be installed, see [https://pnpm.io/installation](https://pnpm.io/installation))
+
+Start by checking out `jazz`
+
+```bash
```

Comment:
We should simplify these instructions. If you're having to reach for `pnpm pack` during an install, that's probably too many steps for someone just wanting to try something out. I'd suggest: 

- `npx degit gardencmp/jazz jazz`
- `cd jazz/examples/chat`
- `pnpm i`
- `pnpm dev`

Reply:
In fact, the current instructions miss the repo build step, so I don't think it'll work.

Reply:
That's much better and it works for me! Thanks

Reply:
I will do it in a separate PR and apply to all the examples